### PR TITLE
AM-2730 Civil mapping changes for staff and judicial team leader roles

### PR DESCRIPTION
**Pull request by prashanthkumarnb**
### JIRA link (if applicable) ###

   [AM-2730](https://tools.hmcts.net/jira/browse/AM-2730) _"Prod Civil role issue - Any user with a team leader role does not inherit the permissions of the staff role that it has responsibility for"_

### Change description ###

Civil mapping changes for staff and judicial team leader roles.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by mikebrownccd — DISMISSED** [review]
LGTM, had walkthrough with Prashanth

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCivilJudicialRoleMappingTest.java [hunk 1]
```diff
@@ -404,5 +405,50 @@ void civilJudicialScenario_DistrictJudgeInRetirement() {
...
+        //Execute Kie session
+        List<RoleAssignment> roleAssignments =
+                buildExecuteKieSession(getFeatureFlags("civil_wa_1_1", true));
+
+        //assertion
+        assertFalse(roleAssignments.isEmpty());
+
+        assertEquals(1, roleAssignments.size());
+        assertEquals(judicialOfficeHolders.stream().iterator().next().getUserId(),roleAssignments.get(0).getActorId());
+        assertEquals(roleAssignments.get(0).getRoleName(),"judge");
```
Comment (mattnayler) [inline src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCivilJudicialRoleMappingTest.java:438]: Check sonar report

File: src/functionalTest/resources/features/F-007/S-061.td.json [hunk 2]
```diff
@@ -70,6 +70,25 @@
             "substantive": "N"
           }
         },
+        {
```
[the remark this answers is not in the document]
Comment (mattnayler) [reply src/functionalTest/resources/features/F-007/S-061.td.json:73]: Resolved in cf2445f5e9247531265605d4a00f30ab1c7b010c in #1428.

File: src/functionalTest/resources/features/F-007/S-082.td.json [hunk 3]
```diff
@@ -92,6 +92,28 @@
             "substantive": "N",
             "contractType": "Salaried"
           }
+        },
+        {
+          "id": "[[ANYTHING_PRESENT]]",
```
[the remark this answers is not in the document]
Comment (mattnayler) [reply src/functionalTest/resources/features/F-007/S-082.td.json:97]: Resolved in cf2445f5e9247531265605d4a00f30ab1c7b010c in #1428.

File: src/main/resources/validationrules/civil/civil-caseworker-mapping.drl [hunk 4]
```diff
@@ -75,6 +75,35 @@ then
       logMsg("Rule : civil_tribunal_caseworker_org_role");
 end;
 
+/*
+ * CIVIL staff "tribunal-caseworker" v11 Org role mapping.
+ */
+
+rule "v11_civil_tribunal_caseworker_org_role"
```
[the remark this answers is not in the document]
Comment (mattnayler) [reply src/main/resources/validationrules/civil/civil-caseworker-mapping.drl:82]: Resolved in 40d755403d077d0c957f2d926962c01c1ac76270 in #1428.

**Pull request merged**